site/zenodo_rdm/openaire/serializers/schema.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""OpenAire schema."""
+
+from invenio_access.permissions import system_identity
+from invenio_communities.proxies import current_communities
+from marshmallow import Schema, fields, missing, pre_dump
+from zenodo_legacy.funders import FUNDER_ACRONYMS, FUNDER_ROR_TO_DOI
+
+from zenodo_rdm.openaire.utils import (
+    get_resource_type_vocabulary,
+    openaire_datasource_id,
+    openaire_original_id,
+)
+
+
+class OpenAIRESchema(Schema):
+    """Schema for records in OpenAIRE-JSON.
+
+    OpenAIRE Schema: https://www.openaire.eu/schema/1.0/oaf-result-1.0.xsd
+    OpenAIRE Vocabularies: http://api.openaire.eu/vocabularies
+    """
+
+    originalId = fields.Method("get_original_id", required=True)
+    title = fields.Str(attribute="metadata.title", required=True)
+    description = fields.Str(attribute="metadata.description")
+    url = fields.Method("get_url", required=True)
+
+    authors = fields.Method("get_authors")
+
+    # TODO language is not being sent on Zenodo legacy. Also, it seems to be incorrect with the OA specs.
```

Comment:
This is important. 

Evidence: 

1) serialization seems incorrect in legacy: 
From their [docs ](https://www.openaire.eu/schema/latest/doc/oaf.html) , `language` is of type `qualifierAttributeGroup` (and not `xs:string`)

2) legacy is not serializing it either:

This [record](https://sandbox.zenodo.org/record/1243324) on legacy sandbox is serialized to: 

```json
{
    "authors": [
        "Test, awasd"
    ],
    "collectedFromId": "opendoar____::2659",
    "description": "<p>Something</p>",
    "hostedById": "opendoar____::2659",
    "licenseCode": "OPEN",
    "originalId": "oai:zenodo.org:1243324",
    "pids": [
        {
            "type": "oai",
            "value": "oai:zenodo.org:1243324"
        },
        {
            "type": "doi",
            "value": "10.5072/zenodo.1243324"
        }
    ],
    "resourceType": "0001",
    "title": "etesaq",
    "type": "publication",
    "url": "https://zenodo.org/record/1243324"
}
```

Note there is no `language` field (neither `version` actually).